fahrschein-http-spring/build.gradle

```diff
@@ -10,4 +10,8 @@ dependencies {
     testImplementation(testFixtures(project(':fahrschein-http-test-support')))
 }
 
+tasks.withType(JavaCompile).configureEach {
+    options.release = 17
```

Comment:
There is also [fahrschein-http-jdk11](https://github.com/zalando-nakadi/fahrschein/tree/main/fahrschein-http-jdk11). What will happen to this module?

Reply:
That module is independent of spring and already now has a baseline of JDK11. So, for projects that don't depend on fahrschein-http-spring nothing changes.